realsense_camera/src/base_nodelet.cpp

```diff
@@ -848,7 +852,9 @@ namespace realsense_camera
             image_[stream_index]).toImageMsg();
 
         msg->header.frame_id = optical_frame_id_[stream_index];
-        msg->header.stamp = ros::Time(camera_start_ts_) + ros::Duration(frame_ts * 0.001); // Publish timestamp to synchronize frames.
+        //msg->header.stamp = ros::Time(camera_start_ts_) + ros::Duration(frame_ts * 0.001); // Publish timestamp to synchronize frames.
```

Comment:
Was this change for a specific reason?  Just hate modifying the driver if we don't have to so we can upgrade without problems.